cli/test/schema.spec.js

```diff
@@ -31,90 +33,83 @@ indexes = ["datetime"]
 template = "any()a;lkdfjlakjf;ladkfjal;kfj"
 `
 
-parse_schema(testSchema)
-const currentOpts = {};
+const valid_file_system = {
+  '.hz': {
+    'schema.toml': testSchema,
+  },
+};
+
+const currentOpts = { };
+
 describe("hz schema", () => {
-  before(() => {
-    // Start RethinkDB server
-    return new Promise((resolve, reject) => {
-      start_rethinkdb_server({
-        driverPort: 38015,
-        bind: ["127.0.0.1"],
-        dataDir: tmpdir() + "/horizon-test",
-        db: "horizon_schema_test"
-      }).then((server) => {
-        currentOpts.dbPort = server.driverPort;
-        resolve();
-      }).catch((err) => {
-        console.log(err)
-      });
-    });
-  });
+  let rdb_server;
+  let rdb_conn;
+
+  before('start rethinkdb', () =>
+    start_rdb_server({
+      bind: ["127.0.0.1"],
+      dataDir: tmpdir() + "/horizon-test",
+    }).then((server) => {
+      rdb_server = server;
+    })
+  );
+
+  after('stop rethinkdb', () => rdb_server.close());
+
+  before('connect to rethinkdb', () =>
+    rdb_server.connect().then((conn) => {
+      rdb_conn = conn;
+    })
+  );
+
+  beforeEach('initialize mockfs', () => mockFs(valid_file_system));
+  afterEach('restore fs', () => mockFs.restore());
 
   describe("hz schema save", () => {
-    
-    beforeEach('set up database state', () => {
-      // Create MockFS
-      mockFs({
-          '.hz': {
-            'schema.toml': testSchema,
-          },
-          'out.toml': '',
-      });
+    //beforeEach('initialize mockfs', () => {
```

Comment:
Ah, I wasn't completely sure if I'd be breaking anything by removing this as I'd added a mockFs initialization higher up in the suite.  Turns out this was no longer necessary, so I removed it.